rgw/v2/tests/s3_swift/configs/test_dbr_high_default_limit.yaml

```diff
@@ -0,0 +1,11 @@
+# test case id: CEPH-83573601
+config:
```

Comment:
can we please update script name here. 
other than that script looks good

Reply:
Done.